builtin/mainmenu/tab_online.lua

```diff
@@ -15,7 +15,7 @@
 --with this program; if not, write to the Free Software Foundation, Inc.,
 --51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
---------------------------------------------------------------------------------
+local server_lookup
```

Comment:
almost forgot: why does this need to be global variable and which purpose does it serve?

Reply:
`server_lookup` is a local variable I introduced. It basically represents the table rows. It is used while generating the table & when to identify selected entries.